pkg/capabilities/consensus/ocr3/aggregators/reduce_test.go

```diff
@@ -517,22 +582,6 @@ func TestReduceAggregator_Aggregate(t *testing.T) {
 					return map[commontypes.OracleID][]values.Value{}
 				},
 			},
-			{
```

Comment:
I removed this because this isn't actually a problem in the current implementation and doesn't error